Automattic/wp-calypso
# Plugin: new onboarding flow adding a plugin

#### Proposed Changes

* This PR creates a new onboarding flow starting from the CTA button to logging in or creating an account and adding the plugin to the checkout page. Project thread: pau2Xa-4j7-p2

#### Testing Instructions



* Choose a plugin - free or paid - from the `/plugins` page as a logged-out user.
* Click on the `Start with this plugin` button to start the new onboarding flow.
  <img width="1041" alt="Yoast_SEO_Plugin_—_WordPress_com_및_Plugin__new_onboarding_flow_adding_a_plugin_by_taggon_·_Pull_Request__67965_·_Automattic_wp-calypso" src="https://user-images.githubusercontent.com/212034/193756999-843eb3ea-cde7-4918-b498-fd3c706b67c5.png">
* On the plans step there should be only two plans: `Business` and `eCommerce`.
  <img width="800" alt="Create_a_site_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/193757674-b2a29053-6ff7-4b64-a86f-18f0bc714239.png">
* Once the checkout is successfully processed you are taken to the plugin installation flow.
* Make sure the new website has both the plan and the plugin.

#### Pre-merge Checklist
